test/functional/atomic/test-atomic-forall-basic.cpp

```diff
@@ -0,0 +1,119 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-19, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+///
+/// Source file containing basic functional tests for atomic operations with forall.
+///
+
+#include "test-atomic-forall-basic.hpp"
+
+// top layer of function templates for tests
+template <typename ExecPolicy, typename AtomicPolicy>
+void testAtomicFunctionPol()
+{
+  testAtomicFunctionBasic<ExecPolicy, AtomicPolicy, int, 10000>();
```

Comment:
Would it be better to use named const variables (or macros even) for these numerical values that are common across test cases.

Reply:
Done.